### ISSUE TITLE
How to render fire stations

### ISSUE THREAD

**Issue by wmisener** [ZeLonewolf/openstreetmap-americana#1036]
Fire stations are one of the last 'community anchor' POIs (#435) for the style to render. There is no one clear iconography associated with them IMO, so I think we could go a number of directions. Therefore, I thought I'd create an issue to discuss possibilities before starting a PR. A few of the options as I see them (further ideas welcome!):

- A classic one used in other OSM maps is a flame icon. This is the approach of e.g. Carto, Maki, and OSMAnd. I find this somehow both intuitive (obviously it's fire) and counter-intuitive (wait, don't firefighters put out fires?!?). It also could be confused with legitimate fire icons we might want in the future, like for firepits.
<img width="282" alt="Screen Shot 2024-02-21 at 9 53 45 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/2a030145-c460-4d76-b129-ffaf233b875b"> <img width="282" src="https://raw.githubusercontent.com/mapbox/maki/main/icons/fire-station.svg">  <img width="282" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/42476c42-7e29-4ef2-a023-a4450f48547f">

- Another option would be a fire truck symbol. This more clearly represents firefighters to me, and would be pretty recognizable iconography. It might be a challenge to get a good looking 20x20px symbol though. One model might be MUTCD W11-8:
<img width="282" src="https://upload.wikimedia.org/wikipedia/commons/f/f5/MUTCD_W11-8.svg">

- One other option that could work is the "Firefighter's cross", something like below. This would have a nice synergy with our badge symbol for police stations. I don't know whether it's widely associated with firefighters outside the US, or recognizable enough on its own. Maybe with the flame in the center?
<img width="282" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/742dbf21-c1c0-4593-83d5-187d00352189"> <img width="282" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/7d59ff03-610e-4e0a-a449-693314d24c63"> 

- Are there other symbols commonly associated with firefighters, but not easily confused with the literal object? Some things that came to mind are hoses, axes, and ladders, but I don't think those would be very clear/distinctive. Maybe those distinctively shaped hats could work?

Anyways, curious what others think.

**Comment by watmildon** [comment]
The first thing that came to my mind was the hat but I wonder if it's distinct enough from other hard hats.

**Comment by 1ec5** [comment]
The examples in my collection are rather uninspiring:

<img width="152" alt="Engels (Huff) Hamilton County 1997" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/f88301e7-8135-4ca5-a8ae-5b37f7630716"> <img width="252" alt="Universal Florida 1995" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/ab370705-8912-463c-ac1b-ba2908dbfc5b"> <img width="268" alt="Rand McNalley Dallas 1994" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/09569135-64d5-402a-a147-d37668d28bf6"> <img width="226" alt="Champion (Mobil) Buffalo 1989" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/69524269-d9ba-414a-89b8-bc6f9ab1b06e"> <img width="290" alt="Verde Valley & Sedona 2022 West Sedona" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/5f712c51-84ee-45a4-b2e1-5aefef51abe4">

I saw these in the road atlas section of a map library, same deal:

<img width="343" alt="Kappa Map Group Austin" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/1c1da460-5251-4725-8e1e-d20332338f33"> <img width="213" alt="Portland Map Book 2015" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/1231218/d7a4ff38-06a4-46fb-b172-84c2438ce9c3">

**Comment by dBitech** [comment]
I think a firetruck  can be made to work.... ![g4](https://github.com/ZeLonewolf/openstreetmap-americana/assets/10306894/f6876f07-a88c-4178-8d3e-ef5656c502ef)
![Fire-Truck](https://github.com/ZeLonewolf/openstreetmap-americana/assets/10306894/7a59b2b0-dd26-482d-9e88-e0748d2fb0d6)

**Comment by adamfranco** [comment]
I've seen photos of firefighters from other parts of the world using significantly [different helmets](https://www.google.com/search?q=firefighter+helmet+types), but the wide sloping back-brim is maybe common enough to be interpreted globally even if it is not the current model in use everywhere.

See also:
 - https://www.fireapparatusmagazine.com/ppe/trends-in-fire-helmet-design/#gref
 - https://www.firerescue1.com/fire-products/personal-protective-equipment-ppe/helmets/articles/code-3-podcast-the-great-helmet-debate-iconic-look-vs-european-design-XtzPiOLkpDCLHney/

That said, I like the little truck too!

**Comment by whubsch** [comment]
I like the truck but I can't quite envision how you could make it discernible at 20 pixels or less. One thought I had was reusing the police badge icon, but with the fire emblem in the middle, something like this:
<img width="217" alt="Screenshot 2024-02-22 at 8 30 49 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/24275736/a003e462-3960-46c8-b116-d4b9b9ef894a">

**Comment by 1ec5** [comment]
> One thought I had was reusing the police badge icon, but with the fire emblem in the middle, something like this:

I’ve seen this emblem a bit as well, always in red alongside a blue police badge icon, mostly on signs but occasionally on maps. We happen to color police station icons in blue because [our color palette](https://github.com/ZeLonewolf/openstreetmap-americana/blob/c3859cec0abf7942171e20b0a3800efc0537d77f/CONTRIBUTING.md#color-scheme-1) assigns blue to any POI in the “Infrastructure” category. Fire stations would also count as infrastructure, but if we take this badge, recolor it in blue, and shrink it down to a POI icon size, I’m not sure it would be easy to distinguish it from a police station.

**Comment by wmisener** [comment]
A few attempts, based on the comments above and [on Slack](https://osmus.slack.com/archives/C01V02K52UX/p1708583235404989):
Police shield with flame: ![poi_fire_station_shield](https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/ce86ea74-b126-4a8f-99f5-7540c8459b82)
<img width="847" alt="Screen Shot 2024-05-07 at 9 55 33 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/14d5fbaf-1c32-454e-aa95-8e8fcea5989d">

Firefighter cross with flame: ![poi_fire_station_cross](https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/57a2ad01-32f3-4437-9c48-d1d9b357c26b)
<img width="847" alt="Screen Shot 2024-05-07 at 10 12 33 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/8186bb96-3db3-4b28-97ba-8a8e391eb0b6">

Fire truck, roughly traced from https://github.com/ZeLonewolf/openstreetmap-americana/issues/1036#issuecomment-1959303565: ![poi_fire_station_truck](https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/abf429a8-9cb4-405b-b336-6338b6d699e6)
<img width="847" alt="Screen Shot 2024-05-07 at 10 30 35 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/2f683185-62d9-4278-9fc4-a57104554a36">

I think the truck might be too finicky/detailed for the style, especially compared to the other "civic" icons. There is some room for the icon to be taller, so I could try that. I feel like making the truck as tall as it is wide will throw off its aspect ratio though. I skipped the hat idea because it seemed harder to draw lol, but I could give it a shot if someone strongly prefers it.

I think either of the first two badges work pretty well! The reused police shield has the pro of increasing style cohesiveness, but the con of making fire stations seem like a subset of police stations. I'd say I'd weakly prefer the cross icon, but would be happy with either.

Opinions and other ideas welcome!

**Comment by wmisener** [comment]
Also, took a photo of a trail map on a hike recently to help me verify some path names, and realized later that I accidentally captured a fire station icon in the wild! It was a firefighter cross style. This was in a California State Park.
![topanga_fire](https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/7a78833f-8860-4c36-b213-c8992411f490)
![topanga_fire_inset](https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/cfb8758e-8e11-4eb1-86cf-0773c574e331)

**Comment by whubsch** [comment]
I like the "cross" one actually. It looks more American to me than the shield. Is there a way to make the external edges a little less wavy/curvy and more rounded like the California State Park map you have a picture of? I think making it a little less ornate might help it fit in better with the other icons, because there's a lot already going on with the flame in the middle.

**Comment by wmisener** [comment]
Good idea, I think this looks better ![poi_fire_station](https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/ad29c5e7-eb6f-4312-92d5-a29b3fc9fc34)

<img width="848" alt="Screen Shot 2024-05-09 at 8 49 46 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/02944331-0b4e-4044-8352-74e3b5da5cd1">

Possible the proportions could still be tweaked, but I think this is a good track! Open for input though

**Comment by ZeLonewolf** [comment]
Until you look really close, the cross looks like one of those diving bell helmets.  I think if you go with the Maltese cross shaped firefighter symbol, you have to drop the flame from it and go with a clean shaped ones.  If you want the flame, I think the badge shape works better.

**Comment by whubsch** [comment]
I have to say I don't see the helmet, but we could go more in the direction of the [FDNY logo](https://simple.wikipedia.org/wiki/New_York_City_Fire_Department#/media/File:Emblem_of_the_New_York_City_Fire_Department.svg), with a divided ring around the flame. A quick mockup (not even centered 😅):

![Screenshot 2024-05-13 at 6 16 54 AM](https://github.com/ZeLonewolf/openstreetmap-americana/assets/24275736/321832d4-c701-43ec-8415-56e94879901b)

**Comment by wmisener** [comment]
Finally found myself some time to come back to this. I can see where the [diving helmet](https://www.robinsdocksideshop.com/Divers_Helmet_BP704AT.jpg) visual impression is coming from, although I can't really say I see it myself. But I can appreciate that maybe the maltese cross with flame has too much going on for its own good. I tried out a version without the flame icon:
<img width="826" alt="Screen Shot 2024-07-05 at 5 54 39 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/d2e8b796-04c1-4e2c-8963-84994cc2b2c4">

Removing the flame also allows one to shrink the circle relative to the "fins", which could be a promising direction:
<img width="826" alt="Screen Shot 2024-07-05 at 5 56 49 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/3fc4b933-e711-4326-9ea8-223ccd275de3">
Although I feel like the proportions of this particular one are somehow not quite right...but maybe that's just me.

I also tried to make the FDNY style one suggested above:
<img width="826" alt="Screen Shot 2024-07-05 at 5 58 23 PM" src="https://github.com/ZeLonewolf/openstreetmap-americana/assets/58491489/a0abafd0-fab8-40be-b087-2e17342febc9">
I think this looks a little like a firepit to me, but I'm open to it. Maybe the ring needs to be thicker?

Now with these and the above shield and flame designs, there are many (too many?) options, fair enough for a relativey hard-to-represent POI. Interested to see which direction people like the most, if any!
